src/main/java/com/social/instagram/service/PostService.java

```diff
@@ -37,7 +39,8 @@ public void updateComment(PostDto postDto) {
     public List<PostResponseDto> getPost(String userId, Pageable pageable) {
         return Stream.of(postRepository.findByUserIdAndFilePathIsNotNull(userId, pageable))
                 .flatMap(Streamable::stream)
-                .map(PostResponseDto::changePostResponseDto)
+                .map(post ->
+                        PostResponseDto.changePostResponseDto(post, postNiceRepository.findByPostId(post.getId())))
```

Comment:
메소드 이름을 `from` 정도로만 지어줘도 의미는 충분히 전달될 것 같습니다~

```suggestion
                        PostResponseDto.from(post, postNiceRepository.findByPostId(post.getId())))
```

Reply:
메소드 이름을 `from`으로 하면 매개변수가 `PostResponseDto`로 변한다는 것을 유추할 수 있어 의미 전달이 된다는 말씀이실까요?

Reply:
네 맞습니다~ 저 데이터로부터 `PostResponseDto`를 생성한다는 의미로 볼 수 있으니까요